pyedflib/edfreader.py

```diff
@@ -126,7 +126,7 @@ def _convert_string(self,s):
         if isinstance(s, bytes):
             return s.decode("latin")
         else:
-            return s.decode("utf-8", "strict")
+            return s.decode("utf_8", "strict")
```

Comment:
should this not be equivalent?

Reply:
Sure, but the reference name for the UTF-8 codec is `utf_8`:
https://docs.python.org/3/library/codecs.html#standard-encodings

The rest are aliases, `utf-8` is not even in the list of aliases:
> **CPython implementation detail:** Some common encodings can bypass the codecs lookup machinery to improve performance. These optimization opportunities are only recognized by CPython for a limited set of (case insensitive) aliases: utf-8, utf8, latin-1, latin1, iso-8859-1, iso8859-1, mbcs (Windows only), ascii, us-ascii, utf-16, utf16, utf-32, utf32, and the same using underscores instead of dashes. Using alternative aliases for these encodings may result in slower execution.